bool check_health
---
string status_message